packages/storybook/src/stories/ino-menu/ino-menu.stories.js

```diff
@@ -5,31 +5,46 @@ import withStencilReadme from '_local-storybookcore/with-stencil-readme';
 import componentReadme from '_local-elements/src/components/ino-menu/readme.md';
 import './ino-menu.scss';
 
+const BTN_ID_CUSTOM_MENU = 'btn-custom-menu';
+const BTN_ID_DIVIDED_MENU = 'btn-divided-menu';
+
+const ID_CUSTOM_MENU = 'custom-menu';
+const ID_DIVIDED_MENU = 'divided-menu';
+
 function subscribeToComponentEvents() {
   // == event block
   const eventHandler = function (e) {
-    const innerText = e.target.innerText;
+    const targetId = e.target.id;
 
-    if (!innerText.startsWith('OPEN')) {
+    if (targetId !== BTN_ID_CUSTOM_MENU && targetId !== BTN_ID_DIVIDED_MENU) {
       return;
     }
 
-    const clickedMenuId = innerText === 'OPEN MENU' ? 'custom-menu' : 'custom-menu-2';
-    const el = document.getElementById(clickedMenuId);
+    const menuEl = document.getElementById(
+      targetId === BTN_ID_CUSTOM_MENU ?
+        ID_CUSTOM_MENU : ID_DIVIDED_MENU
+    );
 
-    if (!el) {
-      return;
-    }
+    if (!menuEl) return;
 
-    const isMenuOpen = el.getAttribute('ino-open');
-    el.setAttribute('ino-open', isMenuOpen === 'false' ? 'true' : 'false');
+    const isMenuOpen = menuEl.getAttribute('ino-open');
+    menuEl.setAttribute(
+      'ino-open',
+      isMenuOpen === 'false' ? 'true' : 'false'
+    );
   };
 
+  function closeHandler(evt) {
+    evt.target.setAttribute('ino-open', 'false');
+  }
+
   document.addEventListener('click', eventHandler);
+  document.addEventListener('menuClose', closeHandler)
 
   // unsubscribe function will be called by Storybook
   return () => {
     document.removeEventListener('click', eventHandler);
```

Comment:
For better code readability I would rename the eventHandler e.g. clickHandler, since there is another eventHandler (closeHandler) now